modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/init.jsp

```diff
@@ -7,33 +7,7 @@
 
 <%@ include file="/init.jsp" %>
 
-<%@ taglib uri="http://liferay.com/tld/learn" prefix="liferay-learn" %><%@
-taglib uri="http://liferay.com/tld/react" prefix="react" %>
+<%@ taglib uri="http://liferay.com/tld/react" prefix="react" %>
 
-<%@ page import="com.liferay.client.extension.exception.ClientExtensionEntryNameException" %><%@
-page import="com.liferay.client.extension.exception.ClientExtensionEntryTypeSettingsException" %><%@
-page import="com.liferay.client.extension.type.CustomElementCET" %><%@
-page import="com.liferay.client.extension.type.FDSCellRendererCET" %><%@
-page import="com.liferay.client.extension.type.FDSFilterCET" %><%@
-page import="com.liferay.client.extension.type.GlobalCSSCET" %><%@
-page import="com.liferay.client.extension.type.GlobalJSCET" %><%@
-page import="com.liferay.client.extension.type.IFrameCET" %><%@
-page import="com.liferay.client.extension.type.JSImportMapsEntryCET" %><%@
-page import="com.liferay.client.extension.type.StaticContentCET" %><%@
-page import="com.liferay.client.extension.type.ThemeCSSCET" %><%@
-page import="com.liferay.client.extension.type.ThemeFaviconCET" %><%@
-page import="com.liferay.client.extension.type.ThemeJSCET" %><%@
-page import="com.liferay.client.extension.type.ThemeSpritemapCET" %><%@
-page import="com.liferay.client.extension.type.annotation.CETProperty" %><%@
-page import="com.liferay.client.extension.web.internal.constants.ClientExtensionAdminFDSNames" %><%@
-page import="com.liferay.client.extension.web.internal.constants.ClientExtensionAdminWebKeys" %><%@
-page import="com.liferay.client.extension.web.internal.display.context.ClientExtensionAdminDisplayContext" %><%@
-page import="com.liferay.client.extension.web.internal.display.context.EditClientExtensionEntryDisplayContext" %><%@
-page import="com.liferay.client.extension.web.internal.display.context.ViewClientExtensionEntryDisplayContext" %><%@
-page import="com.liferay.learn.LearnMessageUtil" %><%@
-page import="com.liferay.portal.kernel.service.WorkflowDefinitionLinkLocalServiceUtil" %><%@
-page import="com.liferay.portal.kernel.util.HashMapBuilder" %>
-
-<%@ page import="java.lang.reflect.Method" %>
-
-<%@ page import="java.util.Collection" %>
\ No newline at end of file
+<%@ page import="com.liferay.learn.LearnMessageUtil" %><%@
+page import="com.liferay.portal.kernel.util.HashMapBuilder" %>
```

Comment:
@kresimir-coko aren't we removing too many page imports here?

Reply:
yeah @dsanz something with my SF is broken, and has been for a little while now

<img src="https://media3.giphy.com/media/PmRgaD2xj0KH2pPrVF/giphy-downsized-medium.gif"/>